src/components/Skeletons/DataOverview.vue

```diff
@@ -488,6 +488,6 @@ export default {
 // some reusable styles
 
 .with-warnings {
-  color: var(--yellow-400);
+  color: var(--yellow-500);
```

Comment:
why did the color change?

Reply:
Because the contrast of the "Warnings" text in the tabs here is really poor with yellow 400

![image](https://user-images.githubusercontent.com/5774638/184366322-8a50ba45-a614-492a-bb2b-e196cd3d4a83.png)

So I took the executive decision to change it by one level:

![image](https://user-images.githubusercontent.com/5774638/184366457-029f9db2-5a4b-4d6f-97c2-13f96d7c906e.png)